### DISCUSSION TITLE
v10.7.0

### DISCUSSION THREAD

**Discussion by cspotcode** [TypeStrong/ts-node#1680] [Releases]
Discussion thread for the v10.7.0 release.

[Release notes](https://github.com/TypeStrong/ts-node/releases/tag/v10.7.0)

**Comment by leotm** [comment]
cheers for https://github.com/TypeStrong/ts-node/issues/1662 ^ both [@swc/core and @swc/wasm](https://github.com/TypeStrong/ts-node#swc) working well with [TSConfig](https://github.com/leotm/react-native-template-typescript/commit/20d0abb6e3e3570897bafd62155d0b79f1673e3c) and pretty speedy

**Reply by leotm** [reply]
maybe for a separate discussion, unsure which worth keeping/using atm ([/core](https://swc.rs/docs/usage/core) or [/wasm](https://swc.rs/docs/usage/wasm))

since perf seems relatively fast in my [minimal template](https://github.com/leotm/react-native-template-typescript)

**Reply by cspotcode** [reply]
That's great, I'm glad to hear it is working well for you. 

Good question about core vs wasm.  I created a separate discussion for it here: #1681 

The short answer is: I recommend trying @swc/core first and switching to @swc/wasm if you hit problems.
